packages/editor/src/components/inputs/EulerInput.tsx

```diff
@@ -20,20 +23,36 @@ type EulerInputProps = {
   unit?: string
 }
 
+//To be set by editor control functions
+export const EulerState = defineState({
```

Comment:
I wouldn't create a hyperflux state for this very specific functionality. Better to just create a normal `hookstate()` state.